sopel/db.py

```diff
@@ -595,28 +648,36 @@ def merge_nick_groups(self, first_nick: str, second_nick: str):
         """
         first_id = self.get_nick_id(first_nick, create=True)
         second_id = self.get_nick_id(second_nick, create=True)
-        session = self.ssession()
-        try:
+        with self.session() as session:
             # Get second_id's values
-            res = session.query(NickValues).filter(NickValues.nick_id == second_id).all()
+            results = session.execute(
+                select(NickValues).where(NickValues.nick_id == second_id)
+            ).scalars()
+
             # Update first_id with second_id values if first_id doesn't have that key
-            for row in res:
-                first_res = session.query(NickValues) \
-                    .filter(NickValues.nick_id == first_id) \
-                    .filter(NickValues.key == row.key) \
-                    .one_or_none()
+            for row in results:
+                first_res = session.execute(
+                    select(NickValues)
+                    .where(NickValues.nick_id == first_id)
+                    .where(NickValues.key == row.key)
+                ).scalar_one_or_none()
```

Comment:
This is just a style thing, but would it make more sense to do .scalars().one_or_none() rather than use the compound function?

Reply:
The doc says it's identical.

Reply:
For sure, but just given we use .scalars() earlier in the code, was just wondering if it made more sense to keep that same style. I don't care either way, just wanted to call it out.

Reply:
OK, I realized what you were talking here: `scalars()` (notice the S) is not the same as `scalar`. Everywhere else where it was possible, I used `scalar_one_or_none()`, and when necessary I used `scalars()` . So no inconsistency here.